azure-defender-easm/add-domain-to-easm/service.py

```diff
@@ -20,7 +19,7 @@
 logger = logging.getLogger()
 
 NAME = os.getenv("NAME", "add-domain-to-easm")
-SUBSCRIBE_TO = os.getenv("SUBSCRIBE_TO", "domains.*")
```

Comment:
This is set in the `deployment.yaml` too so we have to make the change there as well

Reply:
Good catch